com.mikrotik.WinBox.yml

```diff
@@ -0,0 +1,44 @@
+id: com.mikrotik.WinBox
+runtime: org.freedesktop.Platform
+runtime-version: "24.08"
+sdk: org.freedesktop.Sdk
+command: winbox
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --device=dri
+  - --share=network
+
+modules:
+  - name: WinBox
+    buildsystem: simple
+    build-commands:
+    - install -Dm755 apply_extra /app/bin/apply_extra
+    - install -Dm755 winbox /app/bin/winbox
+    - install -Dm644 winbox.desktop /app/share/applications/${FLATPAK_ID}.desktop
+    - install -Dm644 ${FLATPAK_ID}.appdata.xml /app/share/metainfo/${FLATPAK_ID}.metainfo.xml
+    - install -Dm644 winbox512.png /app/share/icons/hicolor/512x512/apps/${FLATPAK_ID}.png
+    sources:
+    - type: script
+      dest-filename: apply_extra
+      commands:
+        - unzip WinBox_Linux.zip
```

Comment:
Can you silence unzip output here ? It messes up the Flatpak CLI when installing.